news/2025/2025-03-02-osumania-7k-world-cup-2025-concludes.md

```diff
@@ -0,0 +1,457 @@
+
+To me, these games are like Minecraft: a sandbox where we can play however we want! For example, we have multiple categories and play styles to choose from, such as LN, rice, hybrid, etc... just to name a few. After all, the main goal of a game is to have fun, especially with friends all around the world. When you're enjoying a game with friends, is there any more compelling reason than to enjoy the moment with them?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/7080063" style="background-image: url('/wiki/shared/avatars/RandomeLoL.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-nat" href="https://osu.ppy.sh/users/7080063">RandomeLoL</a></p>
+
+It's hard to ignore all the changes that have taken place this year. How does the team feel about these changes, in particular the change to the mappool format? What about the addition of Round of 32, has this made the tournament ever so slightly more competitive?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/7304158" style="background-image: url('/wiki/shared/avatars/Crusher.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/7304158">Crusher</a></p>
+
+In my opinion, the tournament has definitely improved from previous years thanks to the addition of Round of 32 and the EX category. These changes made the tournament more competitive and interesting. In fact, these were quite attractive and necessary to players. As the length of the tournament is longer than last year's, there were more things to see. The production value of the tournament has increased, and the addition of Round of 32 gave more players a chance to play in MWC compared to before. Ultimately, I believe this was the key to make MWC 7K as big as it was this year. The pools need to adapt to the players' rapid increase of skill, which is why I think the addition of EX bracket was a good choice! I believe trying out new things will always be something positive for the tournament.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/758406" style="background-image: url('/wiki/shared/avatars/dressurf.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/758406">dressurf</a></p>
+
+I believe extending the bracket to the Rround of 32 provided a fresh and exciting atmosphere for the event. If more people participate in the MWC in the future, it seems fine keeping up the Round of 32 next year too. It would be a net positive to improve the tournament's atmosphere. My only nitpick was that the EX mappool didn't really feel as hard as the name may've suggested. Other than that, I'm really thankful for the team's efforts this year by providing such great mappools! I had a great time playing them all.
```

Comment:
```suggestion
I believe extending the bracket to the Round of 32 provided a fresh and exciting atmosphere for the event. If more people participate in the MWC in the future, it seems fine keeping up the Round of 32 next year too. It would be a net positive to improve the tournament's atmosphere. My only nitpick was that the EX mappool didn't really feel as hard as the name may've suggested. Other than that, I'm really thankful for the team's efforts this year by providing such great mappools! I had a great time playing them all.
```